starter_code/app.js

```diff
@@ -14,5 +14,18 @@ app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
```

Comment:
This is the way it is supposed to be (at least one of the way) : 

```js

app.get("/beers", (req, res, next) => {
  punkAPI
    .getBeers()
    .then(beers => {
      let beer = beers;
      res.render("beers", { beer });
    })
    .catch(error => {
      console.log(error);
    });
});

```

Reply:
You need to take time on the lessons to really understand how to build layout & use them properly